Java/src/com/copilot/yagl4j/GamePadAxis.java

```diff
@@ -6,48 +6,51 @@
  * @author Mark Lefering
  */
 public enum GamePadAxis {
-    SDL_CONTROLLER_AXIS_INVALID(-1),
-    SDL_CONTROLLER_AXIS_LEFTX(0),
-    SDL_CONTROLLER_AXIS_LEFTY(1),
-    SDL_CONTROLLER_AXIS_RIGHTX(2),
-    SDL_CONTROLLER_AXIS_RIGHTY(3),
-    SDL_CONTROLLER_AXIS_TRIGGERLEFT(4),
-    SDL_CONTROLLER_AXIS_TRIGGERRIGHT(5);
+  SDL_CONTROLLER_AXIS_INVALID(-1),
+  SDL_CONTROLLER_AXIS_LEFTX(0),
+  SDL_CONTROLLER_AXIS_LEFTY(1),
+  SDL_CONTROLLER_AXIS_RIGHTX(2),
+  SDL_CONTROLLER_AXIS_RIGHTY(3),
+  SDL_CONTROLLER_AXIS_TRIGGERLEFT(4),
+  SDL_CONTROLLER_AXIS_TRIGGERRIGHT(5);
 
-    /// Value of the enumeration.
-    private final int value;
+  /// Value of the enumeration.
+  private final int value;
 
-    /**
-     * Constructor of the ControllerAxis enum.
-     *
-     * @param value The value of the controller axis.
-     */
-    GamePadAxis(int value) {
-        // Setting value
-        this.value = value;
-    }
+  /**
+   * Constructor of the ControllerAxis enum.
+   *
+   * @param value The value of the controller axis.
+   */
+  GamePadAxis(int value) {
```

Comment:
i'm guessing these should be explicitly public

Reply:
now it's package private right?

Reply:
You guys know this is the only possible scope (implicitly private) for enum constructors in java?

Reply:
ssst